backend test/test.js

```diff
@@ -0,0 +1,152 @@
+const chai = require('chai');
+var expect  = require('chai').expect;
+
+chai.use(require('chai-http'));
+
+var request = require('request');
+
+describe('Main Page', function() {
+    this.timeout(5000); // How long to wait for a response (ms)
+  
+
+it('Main page body', function(done) {
+   
```

Comment:
as per convention, "it" block should start with a verb such as "should" or "checks", etc. 
I guess it's ok for now since we're on a time crunch, but we'll streamline all tests later so make sure they read well.